common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1232,6 +1235,31 @@
                     });
                 }
             },
+            handleTranscriptCookie: function() {
+                if ($.cookie('show_transcript') === null) {
+                    return;
+                }
+                if ($.cookie('show_transcript') !== 'false') {
+                    this.state.hideCaptions = false;
+                    // keep it going until turned off or in case of null initially change to true
+                    this.updateTranscriptCookie(true);
+                } else {
+                    this.state.hideCaptions = true;
+                }
+                this.hideCaptions(this.state.hideCaptions, true, true);
+            },
+            updateTranscriptCookie: function(method) {
```

Comment:
As discussed in standup, this looks good, just change the name `method` to something more meaningful.